bin/node/src/aleph_cli.rs

```diff
@@ -1,11 +1,27 @@
 use aleph_primitives::DEFAULT_UNIT_CREATION_DELAY;
 use clap::Parser;
 use finality_aleph::UnitCreationDelay;
+use std::path::PathBuf;
 
 #[derive(Debug, Parser, Clone)]
 pub struct AlephCli {
     #[clap(long)]
     unit_creation_delay: Option<u64>,
+
+    /// The directory to save created units to for crash recovery purposes.
+    ///
+    /// Units created by the node are saved under this directory. When restarted after a crash,
+    /// previously-created units are read back from this directory first, helping prevent
+    /// auto-forks. The layout of the directory is unspecified.
+    #[clap(
+        long,
+        value_name = "PATH",
+        required_unless_present("unit-saving"),
+        conflicts_with("unit-saving")
+    )]
+    unit_saving_path: Option<PathBuf>,
+    #[clap(long = "no-unit-saving", parse(from_flag = ignore))]
```

Comment:
now I can see that this combo of parsing and attributes prevent you from launching binary in any configuration :(

my advice would be to get rid of `no-unit-saving` and since the path is optional, it should be enough to leave it alone

Reply:
That's weird. I'm sure it worked before the rebase (in e5f45a4). I'll dig into it further, but do you have any idea what might have happened?

Reply:
I talked to @timorl about what the default should be, and we decided that there actually shouldn't be a default. So the intent here is to force you to either specify a `--unit-saving-path`, or to explicitly turn it off with `--no-unit-saving`.

Reply:
I would suspect duo of `ignore` function and `conflicts_with`, but that's just a guess

Reply:
In case there is no backup, the restart will most likely be not possible. We use initial collection in AlephBFT, and if it is not consistent with the backup, alephbft exits. I think user should be strongly notified that without backup, restart will most likely fail or user should not be able to run node without this flag

Reply:
In case there is no backup, the restart will most likely be not possible. We use initial collection in AlephBFT, and if it is not consistent with the backup, alephbft exits. I think user should be strongly notified that without backup, restart will most likely fail or user should not be able to run node without this flag

Reply:
fair enough; therefore, if this actually works (I had problems myself, but maybe I did it incorrectly), I would insist only on the whitespace and investigating `skip` attribute :)